cpg_workflows/stages/rd_combiner.py

```diff
@@ -704,7 +799,17 @@ def expected_outputs(self, dataset: Dataset) -> dict[str, str | Path]:
         Expected to generate a Seqr index, which is not a file
         """
         sequencing_type = config_retrieve(['workflow', 'sequencing_type'])
-        index_name = f'{dataset.name}-{sequencing_type}-{get_workflow().run_timestamp}'.lower()
+        if only_families := config_retrieve(['workflow', dataset.name, 'only_families'], []):
+            family_sg_data = query_for_family_sequencing_groups(
+                dataset.name,
+                config_retrieve(['workflow', 'sequencing_type']),
+                only_families,
+            )
+            n_sgs = len(family_sg_data['family_sg_ids'])
+            n_families = len(family_sg_data['only_family_ids'])
+            index_name = f'{dataset.name}-{sequencing_type}-{n_sgs}_sgs-{n_families}_families-{get_workflow().run_timestamp}'.lower()
```

Comment:
likewise here, including some fragment of a hash in case we ever want to export multiple families separately from the same callset